### PR TITLE
Change presentation guidelines to explicit declaration

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

File: governance/presentations.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# STAG presentations
+
+Part of the STAG activities include having guest presentations by members of the community. We welcome any topic related to our mission and charter. Typical topics include projects, real-world use-cases, challenges or success stories. However, presentations must follow the following guielines.
```
Comment (ashutosh-narkar) [inline governance/presentations.md:3]: Typo `guielines`

File: governance/presentations.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Guidelines
+
+The presentation must follow the guidelines:
```
Comment (ashutosh-narkar) [inline governance/presentations.md:7]: Nit: We probably don't need this line.

**Review by ashutosh-narkar — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/presentation.md [hunk 3]
```diff
@@ -42,3 +19,4 @@ Availability: What is the availability times of the speakers to present the topi
 TO DO
 - [ ] SIG Representative
```
Comment (TheFoxAtWork) [inline .github/ISSUE_TEMPLATE/presentation.md:20]:
```suggestion
- [ ] TAG Representative
```

File: governance/presentations.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# STAG presentations
```
Comment (TheFoxAtWork) [inline governance/presentations.md:1]:
```suggestion
# Security TAG presentations
```

File: governance/presentations.md (hunk 1 above)
Comment (TheFoxAtWork) [reply governance/presentations.md:3]:
```suggestion
Part of the STAG activities include having guest presentations by members of the community. 
We welcome any topic related to our mission and charter. Typical topics include projects, 
real-world use-cases, challenges or success stories. However, presentations must follow the 
following guidelines.
```

File: governance/presentations.md (hunk 2 above)
Comment (TheFoxAtWork) [inline governance/presentations.md:7]:
```suggestion
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
Suggestions made.

**Comment by lumjjb** [comment]
Suggestions applied!

File: governance/presentations.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Part of the STAG activities include having guest presentations by members of the community. 
+We welcome any topic related to our mission and charter. Typical topics include projects, 
+real-world use-cases, challenges or success stories. However, presentations must follow the 
+following guidelines.
+
+## Guidelines
+
+- Presentations are encouraged to expose the SIG to cloud native open source projects, cloud native security concepts, and other cloud native or security groups.
+- Presentations should fit with our charter: https://github.com/cncf/sig-security/blob/master/governance/charter.md
```
Comment (TheFoxAtWork) [inline governance/presentations.md:11]:
```suggestion
- Presentations should fit with [our charter](https://github.com/cncf/tag-security/blob/master/governance/charter.md)
```

**Review by TheFoxAtWork — APPROVED** [review]
LGTM!

**Pull request merged**